manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -81,10 +85,64 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch", "patch"]
+- apiGroups: ["topology.node.k8s.io"]
+  resources: ["noderesourcetopologies"]
+  verbs: ["*"]
+# resources need to be updated with the scheduler plugins used
 - apiGroups: ["scheduling.sigs.k8s.io"]
   resources: ["podgroups", "elasticquotas"]
   verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: topo-aware-scheduler-as-kube-scheduler
```

Comment:
Why we need to treat NodeResourceTopology separately?

Reply:
I wanted to indicate that this binding is only used for the `NodeResourceTopology`, otherwise, it can be removed

Reply:
Could you please try remove it? I don't think `NodeResourceTopology` needs a dedicated RBAC setting.